seatunnel-connectors/seatunnel-connectors-flink-list/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.seatunnel</groupId>
+        <artifactId>seatunnel-connectors</artifactId>
+        <version>${revision}</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>seatunnel-connectors-flink-list</artifactId>
+
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>seatunnel-connectors-flink-list-current</module>
+        <module>seatunnel-connectors-flink-list-previous</module>
```

Comment:
Can we only use one Flink connector module to store all connectors? It seems we will not rely on the `seatunnel-flink-connectors` module in our project, so there will not occur class conflict. 
In example module, we can directly set the target plugin module.
What I concern is that these two module is not clear, and it doesn't solve the conflict problem, if we need to support another connector version, we need to add another module `seatunnel-connectors-flink-list-xx`.

Reply:
This constructure to split two type connector: new version and old version. So we can move it to two different directory. Then starter will only to load jar in new version directory. In example module to add list-current as dependency just make user run easier

Reply:
In my knowledge, the starter will not need to rely on the connector module after your change.

Reply:
Yes, but the starter will search jar from directory, the two different module to identify which directory( connectors or opt ) the connector jar should move to.

Reply:
Ok, I just concern the module name will make user confuse.

Reply:
> Ok, I just concern the module name will make user confuse.

We will add doc to developer start.